cura/PrinterOutput/NetworkedPrinterOutputDevice.py

```diff
@@ -147,6 +147,9 @@ def _createEmptyRequest(self, target: str, content_type: Optional[str] = "applic
         request.setHeader(QNetworkRequest.UserAgentHeader, self._user_agent)
         return request
 
+    def createFormPart(self, content_header: str, data: bytes, content_type: Optional[str] = None) -> QHttpPart:
```

Comment:
That breaks existing code, as in some places it was called privately....
As this could also be done in plugins, we should never break an existing interface, even though it was supposed to be public in the first place.

Reply:
protected/private, what's in the name. Anyways it was called from outside the class even though it has an underscore.

Reply:
Not anymore though because I fixed it in this PR, but we don't know if any plugins are doing it. Although we could argue that the next release has breaking changes anyways, we can break this as well?